blocks/test/full-content.js

```diff
@@ -95,13 +95,23 @@ describe( 'full post content fixture', () => {
 
 	fileBasenames.forEach( f => {
 		it( f, () => {
+			/**
```

Comment:
I think the WordPress standard was written before JSDoc came into wide usage.

I don't really have a preference here, do you?  I just wrote them that way so they would visually look like separate sections.